src/Compilers/Core/MSBuildTask/ManagedToolTask.cs

```diff
@@ -177,5 +180,43 @@ protected static ITaskItem[] GenerateCommandLineArgsTaskItems(List<string> comma
 
             return items;
         }
+
+        private string GetToolDirectory()
```

Comment:
This method could be `static` if `IsSdkFrameworkToCoreBridgeTask` is also `static`.